frontend/__tests__/src/components/Metrics/MetricsStepper/MetricsStepper.test.tsx

```diff
@@ -37,7 +37,7 @@ import {
   updateDeploymentFrequencySettings,
   updateTreatFlagCardAsBlock,
 } from '@src/context/Metrics/metricsSlice'
-import { exportToJsonFile } from '@src/utils/util'
+import { exportToJsonFile, findCaseInsensitiveType } from '@src/utils/util'
```

Comment:
See the GitHub Actions check in the PR, remove the unused variable

Reply:
done